openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -37,54 +32,6 @@ def process(self, instance):
             self.log
         )
 
-        preset = lib.load_capture_preset(data=capture_preset)
-
-        # "isolate_view" will already have been applied at creation, so we'll
-        # ignore it here.
-        preset.pop("isolate_view")
-
-        override_viewport_options = (
-            capture_preset["Viewport Options"]["override_viewport_options"]
-        )
-
-        preset["camera"] = camera
-        preset["start_frame"] = instance.data["frameStart"]
-        preset["end_frame"] = instance.data["frameStart"]
-        preset["camera_options"] = {
-            "displayGateMask": False,
-            "displayResolution": False,
-            "displayFilmGate": False,
-            "displayFieldChart": False,
-            "displaySafeAction": False,
-            "displaySafeTitle": False,
-            "displayFilmPivot": False,
-            "displayFilmOrigin": False,
-            "overscan": 1.0,
-            "depthOfField": cmds.getAttr("{0}.depthOfField".format(camera)),
-        }
```

Comment:
Did we lose this in the refactor to `lib`? This behavior seemed to differ here compared tot the logic in Extract Playblast, right?

Reply:
yes, it seems the camera options look a bit different from extract playblast. We can do the code below knowing that we can include camera options in preset after calling the library:
```
preset["camera_options"].update( {
            "displayGateMask": False,
            "displayResolution": False,
            "displayFilmGate": False,
            "displayFieldChart": False,
            "displaySafeAction": False,
            "displaySafeTitle": False,
            "displayFilmPivot": False,
            "displayFilmOrigin": False,
            "overscan": 1.0,
            "depthOfField": cmds.getAttr("{0}.depthOfField".format(camera)),
        }
```